src/acl_device.cpp

```diff
@@ -642,6 +642,16 @@ CL_API_ENTRY cl_int CL_API_CALL clGetDeviceInfoIntelFPGA(
     RESULT_BITFIELD(0);
   } break;
 
+#ifdef CL_VERSION_3_0
```

Comment:
Hi Sophie,

For my own learning, can you please explain why `ifdef CL_VERSION_3_0` is needed here? Where will this be defined in our flow?

Thanks and best regards,

Zibai

Reply:
I added this because the [atomic fence enum](https://github.com/intel/fpga-runtime-for-opencl/blob/main/include/CL/cl.h#L413) definition is guarded by `ifdef CL_VERSION_3_0`, but I see that `cl_mem_properties` is also guarded, so maybe I'll remove the `ifdef` here.

Reply:
I will think about this and we can discuss further offline.